bundle-size/app.js

```diff
@@ -524,57 +524,9 @@ module.exports = app => {
     response.end();
   });
 
-  v0.post('/commit/:headSha/report', async (request, response) => {
-    const {headSha} = request.params;
-    // TODO(#142): restore the default bundleSize field name here once the
-    // ampproject/amphtml runner starts reporting Brotli sizes in the bundleSize
-    // field.
-    const {baseSha, brotliBundleSize: bundleSize} = request.body;
-
-    if (typeof baseSha !== 'string' || !/^[0-9a-f]{40}$/.test(baseSha)) {
-      return response
-        .status(400)
-        .end('POST request to /report must have commit SHA field "baseSha"');
-    }
-    if (typeof bundleSize !== 'number') {
-      return response
-        .status(400)
-        .end(
-          'POST request to /report must have numeric field "brotliBundleSize"'
-        );
-    }
-
-    const check = await getCheckFromDatabase(headSha);
-    if (!check) {
-      return response
-        .status(404)
-        .end(
-          `${headSha} was not found in bundle-size database; try to rebase ` +
-            'this pull request on the latest commit in the `master` to fix this'
-        );
-    }
-
-    let reportSuccess = await tryReport(check, baseSha, bundleSize);
-    if (reportSuccess) {
-      response.end();
-    } else {
-      response.status(202).end();
-      let retriesLeft = RETRY_TIMES - 1;
-      do {
-        app.log(`Will retry ${retriesLeft} more time(s) in ${RETRY_MILLIS} ms`);
-        await sleep(RETRY_MILLIS);
-        retriesLeft--;
-        reportSuccess = await tryReport(
-          check,
-          baseSha,
-          bundleSize,
-          /* lastAttempt */ retriesLeft == 0
-        );
-      } while (retriesLeft > 0 && !reportSuccess);
-    }
-  });
-
-  v0.post('/commit/:headSha/report.json', async (request, response) => {
+  // TODO(danielrozenberg): remove the .json optional endpoint once the amphtml
```

Comment:
Is this TODO still relevant?

Reply:
Yes, I want to remove the optional `.json` extension from this endpoint so eventually there will be only one copy of it